### ISSUE TITLE
Mailchimp for WP form doesn't display translated error messages

### ISSUE THREAD

**Issue by mudakonn** [ibericode/mailchimp-for-wordpress#284]
Hi,

I connected my Mailchimp account with Wordpress theme (with API key) and added a simple opt-in form in footer. I also translated everything under "Messages" tab.

Somehow when I am now testing my opt-in form, it doesn't display any error messages in the language I translated, but in English. When I check "Messages" tab, it shows everything is translated, but on page it doesn't work.

Is it some kind of a bug or I have done something wrong myself?

By the way, when I fill the form properly and click "Submit", it nicely adds customer to my Mailchimp list. The only problem is with translations.

Best wishes,
Shannes

**Comment by dannyvankooten** [comment]
Hi @mudakonn ,

Do you have a link to your site where we can see this happen? Also, are these the error messages that pop-up as you leave a field empty or the messages that show up after you submit the form?

Thanks!

**Comment by mudakonn** [comment]
Hi,

The site is: http://www.desala.ee

The error messages pop-up when I leave a field empty or fill it incorrectly.

**Comment by dannyvankooten** [comment]
Hi @mudakonn ,

This is actually intended behaviour which requires a bit of explanation.

The first error is when you leave the field empty or enter anything that is clearly not an email address. This validation is handled by the browser (HTML5) and therefore, the message will always be in the same language as the browser of your visitor (which is great in most cases).

**browsers error:**
![image](https://cloud.githubusercontent.com/assets/885856/14486240/1c3788d4-015c-11e6-8968-a0320cf305bc.png)


The second error is when all form fields pass browser validation and the plugin performs additional server-side validation checks to really make sure the input is valid. For example, `asd@asd` usually passes browser validation but is obviously not a correct email address. That is when the plugin will show the error messages you configured in your form.

**plugin error (your message**
![image](https://cloud.githubusercontent.com/assets/885856/14486260/2e529d10-015c-11e6-9de5-4a1fcf19d54f.png)

In short: the pop-up bubbles you're seeing are in the language of your visitor already. It is very hard to change these messages and I'd argue that it is best to leave them as they are.

That said, is it definitely possible to disable browser validation altogether, but I generally do not recommend doing that as it would require a form to be submitted before any errors with the input show up.

Hope that helps. If not, let me know!

**Comment by mudakonn** [comment]
Thanks for quick answer, it helped! I will leave it as it is.
